controllers/articles.controller.js

```diff
@@ -24,7 +28,27 @@ function getArticles(req, res, next) {
         .catch(next);
 };
 
+function patchArticleVotes(req, res, next) {
+    const { article_id } = req.params;
+    const { inc_votes } = req.body;
 
+    if (isNaN(article_id)) {
+        return res.status(400).send({ msg: 'Invalid article_id' });
+    }
+
+    if (typeof inc_votes !== 'number') {
+        return res.status(400).send({ msg: 'Bad request: inc_votes must be a number' });
+    }
```

Comment:
You shouldn't need this code here, as PSQL will throw an error for both situations. You can therefore handle the in error handling middleware 👍